### ISSUE TITLE
Load hr/core using addPlugin before starting HHM

### ISSUE THREAD

**Issue by saviola777** [morko/haxroomie#94]
Currently you inject the core plugin [here](https://github.com/morko/haxroomie/blob/master/packages/haxroomie-core/src/room/components/RoomOpener.js#L333) after the user plugins have been loaded, it would be cleaner to add them using `HHM.manager.addPlugin` before calling `HHM.manager.start`. It is also not necessary to poll for HHM to start loading, since there is an event and deferred for that [here](https://github.com/saviola777/haxball-headless-manager/blob/master/src/classes/PluginManager.js#L1217): `HHM.deferreds.managerStarted` and `HHM.events.USER_PLUGINS_LOADED`.
